api/envoy/extensions/filters/http/custom_response/v3/policies.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.custom_response.v3;
+
+import "envoy/config/core/v3/base.proto";
+import "envoy/config/core/v3/extension.proto";
+import "envoy/config/core/v3/substitution_format_string.proto";
+
+import "google/protobuf/wrappers.proto";
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.custom_response.v3";
+option java_outer_classname = "PoliciesProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/custom_response/v3;custom_responsev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: Custom Response Policies]
+// [#extension: envoy.filters.http.custom_response]
+
+// Custom response policy to redirect the original response to a different
+// upstream.
+// [#next-free-field: 7]
+message RedirectPolicy {
+  // The host that will serve the custom response.
+  //
+  // Example:
+  //
+  // .. code-block:: yaml
+  //
+  //    uri: https://www.mydomain.com
+  //
+  string host = 1 [(validate.rules).string = {min_len: 1}];
```

Comment:
What was the reason for splitting redirect URI into host and path?

Reply:
This to mirror how we expect customers to specify custom response policies. They will have a mapping of one backend service/bucket to several different paths for custom responses. Also having this separation is also helpful if the route is matched based on custom headers (with the information in host field) for backend buckets.

Reply:
@paul-r-gall